packages/tracy-client/tracy-client.0.1/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Client bindings to the Tracy profiler (v0.9.1)"
+maintainer: ["Simon Cruanes"]
+authors: ["Bartosz Taudul" "Simon Cruanes"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/imandra-ai/ocaml-tracy"
+bug-reports: "https://github.com/imandra-ai/ocaml-tracy/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "trace"
```

Comment:
```suggestion
  "trace" {>= "0.1"}
```

please consider adding a lower bound

Reply:
it's the lowest version, is there an issue?

Reply:
Oh, indeed. I was following opam-repository's [PR review guideline](https://github.com/ocaml/opam-repository/wiki/PR-checks).

I guess it's not going to cause a problem in this case in general, but there is a technical possibility when an incompatible version with a lower version number (e.g. `0.0.1`?) is published on OPAM. And I guess it's good to add it here so we do not have to double check whether it indeed depends on the lowest version.

(Maybe I'm missing something like if the lower bound is not supplied CI assume the lowest version for that specific package as the lowest bound. In this case it is indeed non-necessary in general)

Actually I think I need help from a more experienced maintainer of opam-repository to settle this.
@avsm can you take a look at this?

Reply:
Great. In this case, everything looks good to me.

Reply:
Just out of curiosity: does the CI assume the lowest version available for a package when doing the lower-bounds test when the lower bound is not explicitly specified in the OPAM file? @mseri

Reply:
The lowest co-installable

Reply:
When it happens that a release breaks some lower bounds, then we go and we fix them. I think so far it only happened once with a widespread package being ported to ocaml 5 or with old packages from before the lower bounds test was added

Reply:
Thanks @haochenx, keeping those guidelines uptodate is really important, so appreciate you clarifying them as well.  Note that there is absolutely no harm in putting an explicit lower bound even if the package is the lowest one -- this guards against the (unlikely) situation where a package is pinned at a lower version than the previous lowest revision.